modules/katello_lifecycle_environment.py

```diff
@@ -75,7 +75,7 @@
   katello_lifecycle_environment:
     username: "admin"
     password: "changeme"
-    server_url: "https://katello.example.com"
+    server_url: "https://foreman.example.com"
```

Comment:
Yeah, I guess that's true. Given that, I'm fine with keeping the `foreman.example.com` urls.